src/coreclr/tools/superpmi/superpmi-shared/methodcontextreader.cpp

```diff
@@ -28,7 +28,7 @@ HANDLE MethodContextReader::OpenFile(const char* inputFile, DWORD flags)
 static std::string to_lower(const std::string& input)
 {
     std::string res = input;
-    std::transform(input.cbegin(), input.cend(), res.begin(), tolower);
+    std::transform(input.cbegin(), input.cend(), res.begin(), (char(*)(char))tolower);
```

Comment:
It seems dangerous to lie about the function signature.

Reply:
I can instead create a wrapper if desired.